### ISSUE TITLE
imagep() missingColor doc is confusing

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1045]
I don't know what this means, and I wrote it:
```R
#' @param  missingColor A colour to be used to indicate missing data, or
#'         \code{NULL} to avoid making the indication.
```

I need to look through the code to see what actually happens. I think it comes out white, with the default value of `NULL`.

**Comment by dankelley** [comment]
No, actually, the following reveals that using `NULL` yields transparent.
```R
library(oce)
v <- volcano # built-in dataset
v[10:15,10:15] <- NA
imagep(v) # white spot where data are NA
# make background red
par(bg='red')
imagep(v,missingCol=NULL) # now that spot is red, showing the col is transparent
```

**Comment by dankelley** [comment]
FIxed in `develop` commit 40e04fda4f598bdc3d90c3aa9e83f52f2666e768
